.buildkite/pipeline.yml

```diff
@@ -10,12 +10,17 @@ common-params:
       - "AWS_SECRET_KEY"
 
 steps:
+  - label: "Publish to Maven Local (to verify build)"
+    command: |
+      npm install
+      ./gradlew publishToMavenLocal
+    plugins:
+      - *docker-container
   - label: "Publish to S3"
     plugins:
       - *docker-container
+    if: build.branch == "trunk"
```

Comment:
I moved this `if` from `buildkite` configuration to be able to execute PR check:
![Screenshot 2024-02-02 at 12 51 29](https://github.com/wordpress-mobile/react-native-libraries-publisher/assets/5845095/fd470dde-771b-4a40-a53b-e2f08bd29652)